### PR TITLE
3.6 CHANGELOG

### PR DESCRIPTION
**Pull request by tobiasd**
The updated CHANGELOG for the 3.6 release of Friendica.

Please have a look for ypots and important things that were forgotten or merged into broader topics you think are too important so they should be listed separately.

I shall be merged as last thing before doing the release.

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -1,3 +1,128 @@
+Version 3.6 (2018-03-xx)
+  Friendica Core:
+    Updates to the translations (DE, EN_GB, EN_US, ES, FR, IT, ZH_CN) [translation teams]
+    Update for the Danish and French regions [Alkarex]
+    Update for the documentation [andyhee, annando, rabuzarus, ratten, rudlof, silke, tobiasd]
+    Updates to the themes [annando, fabrixxm, hoergen, rebeka-catalina, rabuzarus]
+    Enhancements to the ARIA support in frio [rabuzarus]
+    Enhancements to the DB handling [nd structure annando]
```
Comment (annando) [inline CHANGELOG]: What does this "nd structure" mean?

File: CHANGELOG [hunk 2]
```diff
@@ -1,3 +1,128 @@
...
+    Fixed a bug that caused Twidere to crash on reload [annando]
+    Fixed a bug in the exported data to the-federation.info [annando]
+    Fixed a bug in URL completion for feed fragments [annando]
+    Fixed a bug in the notification system about new registrations [annando]
+    Fixed the display of dislikes [annando]
+    Fixed the display of orphand childs in threads [hypolite]
+    Fixed some SQL problems [annando]
+    Fixed the CLI config script [tobiasd]
+    Fixed the forum selection on the network display [annando]
+    Fixed a bug during the import of account [annando]]
```
Comment (annando) [inline CHANGELOG]: There is one "]" too much.

File: CHANGELOG [hunk 3]
```diff
@@ -1,3 +1,128 @@
...
+    Added support of nodeinfo 2.0 [annando]
+    Removed the long deprecated internal templating engine [annando]
+    Removed the obsolete mysql support, you have to use MySQLI or PDO [annando]
+    Removed the unused mood module [annando]
+    Removed connect link from side panel when it should not be there [annando]
+    Removed very old updating routines [annando]
+    Dependencies are now (mostly) handled by composer [hypolite, zeroadam]
+    General code refactoring and beautification work [annando, hypolite, tobiasd, zeroadam]
+    ejabberd logs are now handled by syslog [annando]
+    Moved the poller script wo the "scripts" directory and renamed it to worker [annando]
```
Comment (annando) [inline CHANGELOG]: "wo"?

File: CHANGELOG [hunk 4]
```diff
@@ -1,3 +1,128 @@
...
+    Threaded display of conversations is now always enabled [annando]
+    Images send to public forums are now always public as well [annando]
+    The DB cleanup option now includes the conversation table [annando]
+    Hash tags now always search locally [annando]
+    Consistend usage of addons (instead of plugins and addons) [zeroadam]
```
Comment (annando) [inline CHANGELOG]: "Consistent naming" should fit better.

File: CHANGELOG (hunk 1 above)
Comment (tobiasd) [reply CHANGELOG]: `s/\[/a/` plus add [ before the name...

File: CHANGELOG [hunk 5]
```diff
@@ -1,3 +1,128 @@
+Version 3.6 (2018-03-xx)
+  Friendica Core:
+    Updates to the translations (DE, EN_GB, EN_US, ES, FR, IT, ZH_CN) [translation teams]
+    Update for the Danish and French regions [Alkarex]
+    Update for the documentation [andyhee, annando, rabuzarus, ratten, rudlof, silke, tobiasd]
+    Updates to the themes [annando, fabrixxm, hoergen, rebeka-catalina, rabuzarus]
+    Enhancements to the ARIA support in frio [rabuzarus]
+    Enhancements to the DB handling and structure [annando]
+    Enhancements to the API [annando, fabrixxm, hypolite, rudlof]
```
Comment (MrPetovan) [inline CHANGELOG]: Can you please mention me by my new GitHub handle, `MrPetovan`? Thanks!

File: CHANGELOG [hunk 6]
```diff
@@ -1,3 +1,128 @@
...
+    Added the possibility to address forums with !forumname [annando]
+    Added option to compare version against upstream version [tobiasd]
+    Added an optional hint that a global community page is global [tobiasd]
+    Added an option to always display the preview image in shared articles even if larger ones exist [annando]
+    Added CLI script to silence accounts on the community page [tobiasd]
+    Added CLI script to block postings to a node from accounts [tobiasd]
+    Added account block interface to the admin panel [hyoplite]
+    Added browser bookmarkled code snipplet [hoergen]
+    Added an additional feature to display a tagcloud on the profile page [rabuzarus]
+    Added retrival of Mastodon server statistics [annando]
```
Comment (MrPetovan) [inline CHANGELOG]: `retrieval`

File: CHANGELOG [hunk 7]
```diff
@@ -1,3 +1,128 @@
...
+    Added Atom feed for conversations [annando]
+    Added the possibility to address forums with !forumname [annando]
+    Added option to compare version against upstream version [tobiasd]
+    Added an optional hint that a global community page is global [tobiasd]
+    Added an option to always display the preview image in shared articles even if larger ones exist [annando]
+    Added CLI script to silence accounts on the community page [tobiasd]
+    Added CLI script to block postings to a node from accounts [tobiasd]
+    Added account block interface to the admin panel [hyoplite]
+    Added browser bookmarkled code snipplet [hoergen]
+    Added an additional feature to display a tagcloud on the profile page [rabuzarus]
```
Comment (MrPetovan) [inline CHANGELOG]: `tag cloud`

File: CHANGELOG [hunk 8]
```diff
@@ -1,3 +1,128 @@
...
+    Fixed a problem with the expiration of accounts [annando]
+    Added Atom feed for conversations [annando]
+    Added the possibility to address forums with !forumname [annando]
+    Added option to compare version against upstream version [tobiasd]
+    Added an optional hint that a global community page is global [tobiasd]
+    Added an option to always display the preview image in shared articles even if larger ones exist [annando]
+    Added CLI script to silence accounts on the community page [tobiasd]
+    Added CLI script to block postings to a node from accounts [tobiasd]
+    Added account block interface to the admin panel [hyoplite]
+    Added browser bookmarkled code snipplet [hoergen]
```
Comment (MrPetovan) [inline CHANGELOG]: `bookmarklet code snippet`

File: CHANGELOG [hunk 9]
```diff
@@ -1,3 +1,128 @@
...
+    Enhancements to the support of Open Graph with images [hoergen]
+    Enhancements to the Diaspora federation (participation signal, relay of dislikes, basic forum support for D*, Birthdays) [annando]
+    Enhancements to the OStatus federation [annando]
+    Enhancements to the handling of feed contacts [hypolite]
+    Enhancements to the display of threaded discussions (optional) [hypolite]
+    Enhancements to the display of events [hoergen]
+    Enhancements to the ACL dialog (selection of forums) [rabuzarus]
+    Enhancements to the handling of new connections [annando]
+    Enhancements to the vitality check of contacts [annando]
+    Enhancements to the deamon script [annando]
```
Comment (MrPetovan) [inline CHANGELOG]: `daemon`

File: CHANGELOG [hunk 10]
```diff
@@ -1,3 +1,128 @@
...
+    Enhancements to the federation stats [annando, tobiasd]
+    Enhancements to the interaction with puclic postings [annando]
```
Comment (MrPetovan) [inline CHANGELOG]: `public posts`

Shouldn't it be an addition?

File: CHANGELOG [hunk 11]
```diff
@@ -1,3 +1,128 @@
...
+    Enhancements to the OEmbed handling [hypolite]
+    Fixed a bug that triggered the display of activities on the cummunity page [annando]
+    Fixed a bug with personal notes [annando]
+    Fixed a display issue of long postings when using the showmore option [annando]
+    Fixed a bug that caused Twidere to crash on reload [annando]
+    Fixed a bug in the exported data to the-federation.info [annando]
+    Fixed a bug in URL completion for feed fragments [annando]
+    Fixed a bug in the notification system about new registrations [annando]
+    Fixed the display of dislikes [annando]
+    Fixed the display of orphand childs in threads [hypolite]
```
Comment (MrPetovan) [inline CHANGELOG]: `orphans` instead of `orphand childs`

File: CHANGELOG [hunk 12]
```diff
@@ -1,3 +1,128 @@
...
+    Added an option to always display the preview image in shared articles even if larger ones exist [annando]
+    Added CLI script to silence accounts on the community page [tobiasd]
+    Added CLI script to block postings to a node from accounts [tobiasd]
+    Added account block interface to the admin panel [hyoplite]
+    Added browser bookmarkled code snipplet [hoergen]
+    Added an additional feature to display a tagcloud on the profile page [rabuzarus]
+    Added retrival of Mastodon server statistics [annando]
+    Added Atom feed that only contains top level postings of a user [annando]
+    Added tag following via saved search for #hashtag [annando]
+    Added PHP version informations to the admin panel [hypolite]
```
Comment (MrPetovan) [inline CHANGELOG]: `information`

File: CHANGELOG [hunk 13]
```diff
@@ -1,3 +1,128 @@
...
+    Removed very old updating routines [annando]
+    Dependencies are now (mostly) handled by composer [hypolite, zeroadam]
+    General code refactoring and beautification work [annando, hypolite, tobiasd, zeroadam]
+    ejabberd logs are now handled by syslog [annando]
+    Moved the poller script to the "scripts" directory and renamed it to worker [annando]
+    Threaded display of conversations is now always enabled [annando]
+    Images send to public forums are now always public as well [annando]
+    The DB cleanup option now includes the conversation table [annando]
+    Hash tags now always search locally [annando]
+    Consistend naming of addons (instead of plugins and addons) [zeroadam]
```
Comment (MrPetovan) [inline CHANGELOG]: `Consistent`

File: CHANGELOG [hunk 14]
```diff
@@ -1,3 +1,128 @@
...
+    Fixed the issue that Atom feeds of forums had no postings listed [annando]
+    Fixed a problem with the expiration of accounts [annando]
+    Added Atom feed for conversations [annando]
+    Added the possibility to address forums with !forumname [annando]
+    Added option to compare version against upstream version [tobiasd]
+    Added an optional hint that a global community page is global [tobiasd]
+    Added an option to always display the preview image in shared articles even if larger ones exist [annando]
+    Added CLI script to silence accounts on the community page [tobiasd]
+    Added CLI script to block postings to a node from accounts [tobiasd]
+    Added account block interface to the admin panel [hyoplite]
```
Comment (fabrixxm) [inline CHANGELOG]: s/hyoplite/hypolite/

File: CHANGELOG (hunk 10 above)
Comment (tobiasd) [reply CHANGELOG]: Viewing is interacting too ;-)

Comment (MrPetovan) [reply CHANGELOG]: Fair enough, the typo remains.

File: CHANGELOG [hunk 15]
```diff
@@ -1,3 +1,128 @@
+Version 3.6 (2018-03-xx)
+  Friendica Core:
+    Updates to the translations (DE, EN_GB, EN_US, ES, FR, IT, ZH_CN) [translation teams]
+    Update for the Danish and French regions [Alkarex]
+    Update for the documentation [andyhee, annando, rabuzarus, ratten, rudlof, silke, tobiasd]
+    Updates to the themes [annando, fabrixxm, hoergen, rebeka-catalina, rabuzarus]
+    Enhancements to the ARIA support in frio [rabuzarus]
+    Enhancements to the DB handling and structure [annando]
+    Enhancements to the API [annando, fabrixxm, MrPetrovan, rudlof]
```
Comment (MrPetovan) [inline CHANGELOG]: `MrPetovan` 🤦

File: CHANGELOG [hunk 16]
```diff
@@ -1,3 +1,128 @@
...
+    Enhancements to the OStatus federation [annando]
+    Enhancements to the handling of feed contacts [MrPetrovan]
+    Enhancements to the display of threaded discussions (optional) [MrPetrovan]
+    Enhancements to the display of events [hoergen]
+    Enhancements to the ACL dialog (selection of forums) [rabuzarus]
+    Enhancements to the handling of new connections [annando]
+    Enhancements to the vitality check of contacts [annando]
+    Enhancements to the daemon script [annando]
+    Enhancements to the federation stats [annando, tobiasd]
+    Enhancements to the interaction with puclic posts [annando]
```
Comment (MrPetovan) [inline CHANGELOG]: `public`

File: CHANGELOG (hunk 15 above)
Comment (tobiasd) [reply CHANGELOG]: hypolite was much better ;-)

**Comment by annando** [comment]
BTW: I just had a look at the other versions. This really will be the version with the most closed issues and new features and bugfixes.

**Comment by MrPetovan** [comment]
You didn't believe it or you just wanted to get the scale of the difference?

**Comment by annando** [comment]
Yes.

**Pull request merged**